### PR TITLE
Add support for playing multiple voice messages

### PR DESCRIPTION
**Pull request by rudigiesler**
If a list of URLs is given instead of a single URL, then play all of the voice messages in order.

### CODE REVIEW

File: vxfreeswitch/voice.py [hunk 1]
```diff
@@ -370,8 +370,11 @@ def send_outbound_message(self, client, message):
 
         if overrideURL is None:
             yield client.output_message("%s\n" % content)
-        else:
+        elif isinstance(overrideURL, basestring):
             yield client.output_stream(overrideURL)
+        elif isinstance(overrideURL, list):
+            for url in overrideURL:
+                yield client.output_stream(url)
```
Comment (hodgestar) [inline vxfreeswitch/voice.py]: Should we add an `else` clause that logs an error if `overrideURL` is something unexpected?

**Comment by hodgestar** [comment]
Left one minor suggestion, otherwise looks good.

**Comment by rudigiesler** [comment]
@hodgestar Ready for review again

**Comment by rudigiesler** [comment]
The parts that aren't being tested are the new logging commands. I can create tests that would cause things to error, but I'm not sure how to assert that things were logged. Suggestions welcome.

**Comment by hodgestar** [comment]
You can use `vumi.tests.utils.LogCatcher` to catch logs in tests. E.g.
```python
with LogCatcher(log_level=logging.WARN) as lc:
    ... do things ...
    [msg] = lc.messages()
    self.assertEqual(msg, "foo")
```

**Comment by rudigiesler** [comment]
@hodgestar thanks. Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**